Update set-up-new-message-encryption-capabilities.md (#7928)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/set-up-new-message-encryption-capabilities.md

   - The default template names may be different from those displayed above. See [Configuring and managing templates for Azure Information Protection](/azure/information-protection/configure-policy-templates) for more.

> 4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and  Run the Test-IRMConfiguration cmdlet to verify that it passes.

Reviewer: ```suggestion
4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and run the Test-IRMConfiguration cmdlet to verify that it passes.
```

> 4.If the test fails with an error message **Failed to acquire RMS templates**, you need to execute the following commands and  Run the Test-IRMConfiguration cmdlet to verify it passes.

Reviewer: ```suggestion
4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and  Run the Test-IRMConfiguration cmdlet to verify that it passes.
```
